src/SIL.XForge.Scripture/ClientApp/src/app/shared/book-chapter-chooser/book-chapter-chooser.component.scss

```
@@ -7,43 +7,28 @@
 // These styles were written by inspecting the elements using developer and then adding rules until it looked right
 #chapter-select ::ng-deep,
 #book-select ::ng-deep {
-  .mat-form-field-wrapper {
-    margin: 0;
-    padding-bottom: 0;
-    .mat-form-field-flex {
-      padding-top: 0;
-    }
-    .mat-form-field-flex:not(:hover) {
-      background-color: transparent;
-    }
+  height: 36px;
+  .mat-mdc-select-arrow-wrapper {
+    margin-left: 5px;
   }
-  .mat-form-field-underline {
-    display: none;
-  }
-  .mat-select-arrow-wrapper {
-    transform: unset;
-  }
-  .mat-form-field-infix {
-    // It appeared there was a transparent border being used as padding
-    border-top-width: 0;
-    padding-top: 12px;
-    padding-bottom: 10px;
-  }
-  .mat-form-field-infix {
+  .mat-mdc-form-field-infix {
+    min-height: initial;
+    padding-top: 7px;
+    padding-bottom: 5px;
     width: initial;
```

...and Reviewable did not make my last comment a suggestion like I wanted. Here's try 2.

```suggestion
    width: initial;
    min-width: 25px; // prevent weird transparent gap between content and scrollbar in Chromium
```